thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -1,56 +1,70 @@
 import Ember from 'ember';
-import { makeIterable, filterObject, filterPrefix, toBaselineUrn, toCurrentUrn } from 'thirdeye-frontend/helpers/utils';
+import { filterObject, filterPrefix, toBaselineUrn, toCurrentUrn } from 'thirdeye-frontend/helpers/utils';
 import EVENT_TABLE_COLUMNS from 'thirdeye-frontend/mocks/eventTableColumns';
 import config from 'thirdeye-frontend/mocks/filterBarConfig';
+import SHA256 from 'cryptojs/sha256';
 
 const ROOTCAUSE_TAB_DIMENSIONS = "dimensions";
 const ROOTCAUSE_TAB_METRICS = "metrics";
 const ROOTCAUSE_TAB_EVENTS = "events";
 
 export default Ember.Controller.extend({
-
-  /**
-   * QueryParams that needs to update the url
-   */
   queryParams: [
-    'granularity',
-    'filters',
-    'compareMode',
-    'anomalyRangeStart',
-    'anomalyRangeEnd',
-    'analysisRangeStart',
-    'analysisRangeEnd'
+    'metricId',
+    'anomalyId',
+    'shareId'
   ],
-  entitiesService: Ember.inject.service('rootcause-entities-cache'), // service
 
-  timeseriesService: Ember.inject.service('rootcause-timeseries-cache'), // service
+  //
+  // services
+  //
+  entitiesService: Ember.inject.service('rootcause-entities-cache'),
+
+  timeseriesService: Ember.inject.service('rootcause-timeseries-cache'),
+
+  aggregatesService: Ember.inject.service('rootcause-aggregates-cache'),
 
-  aggregatesService: Ember.inject.service('rootcause-aggregates-cache'), // service
+  breakdownsService: Ember.inject.service('rootcause-breakdowns-cache'),
 
-  breakdownsService: Ember.inject.service('rootcause-breakdowns-cache'), // service
+  ajax: Ember.inject.service(),
```

Comment:
Also, any reasons we're using ajax here instead of `fetch`? I don't mind using ajax if it has better support and use case. I just want us to be consistent in our usage.

Reply:
is fetch meant to POST requests?